### DISCUSSION TITLE
Cannot get property from data bucket

### DISCUSSION THREAD

**Discussion by artronen** [mockoon/mockoon#1029] [Help] [answered]
Hello!
I want to get value from data bucket
my bucket named 'mocke': 
```
{
  "name" = "{{faker 'name.firstName'}}",
  "id" = 2
}
```

and I want to use template kind of
```
{
  "name" = {{data 'mocke' 'name'}}
}
```
i expected this 
```
{
  name = "Armand"
}
```
but got full bucket
```
{
  name = {
  "name" = "Armand",
  "id" = 2
}
}
```

**Comment by 255kb** [comment, marked as answer]
Hi, 

You can get data from a bucket only if it is valid JSON, but yours is not. Properties and values should be separated by a colon, not an equal:

```
{
  "name": "{{faker 'name.firstName'}}",
  "id": 2
}
```

**Reply by artronen** [reply]
holycrap, thx!
